.github/workflows/ci.yml

```diff
@@ -11,10 +11,10 @@ jobs:
         uses: actions/checkout@v2
       - name: Compile and test
         id: ci
-        uses: ignition-tooling/action-ignition-ci@master
+        uses: ignition-tooling/action-ignition-ci@bionic
```

Comment:
do we want to use bionic on main or keeping  using main/mater from actions-ignition-ci?

Reply:
We shouldn't use the `master` branch of that repository, we should delete that branch once no more libraries are using it.

The branches to be used are `bionic` and `focal`.